pkg/operator/ceph/object/spec.go

```diff
@@ -111,22 +111,34 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 
 	// Replace default unreachable node toleration
 	k8sutil.AddUnreachableNodeToleration(&podSpec)
-
 	// Set the ssl cert if specified
-	if c.store.Spec.Gateway.SSLCertificateRef != "" {
+	if c.store.Spec.Gateway.SecurePort != 0 {
```

Comment:
Is this more correct or more readable as `IsTLSEnabled()`?

Reply:
Re-asking this question. https://github.com/rook/rook/pull/7598/files#r617695635

Reply:
I want to return an error if they specify `secure port` and didn't give an option for TLS certificates